scripts/zones/The_Garden_of_RuHmet/mobs/Jailer_of_Faith.lua

```diff
@@ -97,17 +110,8 @@ entity.onMobFight = function(mob)
 end
 
 entity.onMobMagicPrepare = function(mob, target, spell)
-    local rnd = math.random()
-    if rnd < 0.5 and mob:hasStatusEffect(xi.effect.MANAFONT) == true then -- quake II replaces existing earth damage spells during manafont
-        return xi.magic.spell.QUAKE_II -- quake II casted exclusively while Manafont is active.
-    elseif rnd < 0.25 then
-        return xi.magic.spell.STONE_IV
-    elseif rnd < 0.5 then
-        return xi.magic.spell.STONEGA_III
-    elseif rnd < 0.75 then
-        return xi.magic.spell.BREAKGA
-    else
-        return xi.magic.spell.SLOWGA
+    if mob:hasStatusEffect(xi.effect.MANAFONT) and math.random() < 0.5 then
+        return xi.magic.spell.QUAKE_II
```

Comment:
might still need to `return spell:getID()` when quake doesn't get hit, not sure.

Reply:
I checked, it falls back to the regular spell list which was identical. Works perfect in testing

Reply:
awesome, thanks Winter